discourse-mixin/dashboards/discourse-jobs.libsonnet

```diff
@@ -0,0 +1,621 @@
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'horizontal',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '9.1.8',
+  targets: [
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: 'count(discourse_rss{type="sidekiq",instance=~"$instance",job=~"$job"})',
+      format: 'time_series',
+      intervalFactor: 2,
+      legendFormat: '',
+      range: true,
+      refId: 'A',
+      step: 40,
+      target: '',
+    },
+  ],
+  title: 'Sidekiq Workers',
+  type: 'stat',
+};
+
+local webWorkersStat = {
+  datasource: promDatasource,
+  description: 'Current number of Web Workers.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [
+        {
+          options: {
+            match: 'null',
+            result: {
+              text: 'N/A',
+            },
+          },
+          type: 'special',
+        },
+      ],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  id: 16,
+  links: [],
+  maxDataPoints: 100,
+  options: {
+    colorMode: 'none',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'horizontal',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '9.1.8',
+  targets: [
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: "count(discourse_rss{type='web',instance=~\"$instance\",job=~\"$job\"})",
+      format: 'time_series',
+      intervalFactor: 2,
+      legendFormat: '',
+      range: true,
+      refId: 'A',
+      step: 40,
+      target: '',
+    },
+  ],
+  title: 'Web Workers',
+  type: 'stat',
+};
+
+local skQueuedStat = {
+  datasource: promDatasource,
+  description: 'Current number of jobs in Sidekiq queue.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [
+        {
+          options: {
+            match: 'null',
+            result: {
+              text: 'N/A',
+            },
+          },
+          type: 'special',
+        },
+      ],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  id: 17,
+  links: [],
+  maxDataPoints: 100,
+  options: {
+    colorMode: 'none',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'horizontal',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
```

Comment:
You could also use max here, rather than in the promql. Net effect is the same, assuming that there aren't several different labels on the job metrics.. Just FYI.